articles/data-factory/data-flow-expression-functions.md

```diff
@@ -16,658 +16,703 @@ ms.date: 02/15/2019
 * ``add(10, 20) -> 30``
 * ``10 + 20 -> 30``
 * ``add('ice', 'cream') -> 'icecream'``
 * ``'ice' + 'cream' + ' cone' -> 'icecream cone'``
-* ``add(toDate('2012-12-12'), 3) -> 2012-12-15 (date value)``
-* ``toDate('2012-12-12') + 3 -> 2012-12-15 (date value)``
-* ``[10, 20] + [30, 40] => [10, 20, 30, 40]``
----
+* ``add(toDate('2012-12-12'), 3) -> toDate('2012-12-15')``
+* ``toDate('2012-12-12') + 3 -> toDate('2012-12-15')``
+* ``[10, 20] + [30, 40] -> [10, 20, 30, 40]``
+* ``toTimestamp('2019-02-03 05:19:28.871', 'yyyy-MM-dd HH:mm:ss.SSS') + (days(1) + hours(2) - seconds(10)) -> toTimestamp('2019-02-04 07:19:18.871', 'yyyy-MM-dd HH:mm:ss.SSS')``
+*********************************
 ### <code>addDays</code>
 <code><b>addDays(<i>&lt;date/timestamp&gt;</i> : datetime, <i>&lt;days to add&gt;</i> : integral) => datetime</b></code><br/><br/>
 Add days to a date or timestamp. Same as the + operator for date
-* ``addDays(toDate('2016-08-08'), 1) -> 2016-08-09``
----
+* ``addDays(toDate('2016-08-08'), 1) -> toDate('2016-08-09')``
+*********************************
 ### <code>addMonths</code>
-<code><b>addMonths(<i>&lt;date/timestamp&gt;</i> : datetime, <i>&lt;months to add&gt;</i> : integral) => datetime</b></code><br/><br/>
-Add months to a date or timestamp
-* ``addMonths(toDate('2016-08-31'), 1) -> 2016-09-30``
-* ``addMonths(toTimestamp('2016-09-30 10:10:10'), -1) -> 2016-08-31 10:10:10``
----
+<code><b>addMonths(<i>&lt;date/timestamp&gt;</i> : datetime, <i>&lt;months to add&gt;</i> : integral, [<i>&lt;value3&gt;</i> : string]) => datetime</b></code><br/><br/>
+Add months to a date or timestamp. You can optionally pass a timezone
+* ``addMonths(toDate('2016-08-31'), 1) -> toDate('2016-09-30')``
+* ``addMonths(toTimestamp('2016-09-30 10:10:10'), -1) -> toTimestamp('2016-08-31 10:10:10')``
+*********************************
 ### <code>and</code>
 <code><b>and(<i>&lt;value1&gt;</i> : boolean, <i>&lt;value2&gt;</i> : boolean) => boolean</b></code><br/><br/>
 Logical AND operator. Same as &&
 * ``and(true, false) -> false``
 * ``true && false -> false``
----
+*********************************
 ### <code>asin</code>
 <code><b>asin(<i>&lt;value1&gt;</i> : number) => double</b></code><br/><br/>
 Calculates an inverse sine value
 * ``asin(0) -> 0.0``
----
+*********************************
 ### <code>atan</code>
 <code><b>atan(<i>&lt;value1&gt;</i> : number) => double</b></code><br/><br/>
 Calculates a inverse tangent value
 * ``atan(0) -> 0.0``
----
+*********************************
 ### <code>atan2</code>
 <code><b>atan2(<i>&lt;value1&gt;</i> : number, <i>&lt;value2&gt;</i> : number) => double</b></code><br/><br/>
 Returns the angle in radians between the positive x-axis of a plane and the point given by the coordinates
 * ``atan2(0, 0) -> 0.0``
----
+*********************************
 ### <code>avg</code>
 <code><b>avg(<i>&lt;value1&gt;</i> : number) => number</b></code><br/><br/>
 Gets the average of values of a column
-* ``avg(sales) -> 7523420.234``
```

Comment:
This shouldn't be deleted

Reply:
Kiran removed, the number doesn't make sense without a datset